tests/fixtures/optimizer/tpc-h/tpc-h.sql

```diff
@@ -1043,6 +1043,7 @@ SELECT
   COUNT(*) AS "custdist"
 FROM (
   SELECT
+    "customer"."c_custkey" AS "c_custkey",
```

Comment:
This isn't really ideal. But right now we support queries like this:

```sql
SELECT 
  x.b, 
  x.a 
FROM x 
LEFT JOIN y 
GROUP BY b
```

"b" is ambiguous, but some dialects might assume "b" is the output column "x.b". At least we have tests that make that assumption...